package.json

```diff
@@ -6,12 +6,14 @@
   "contributors": [
     "edqx"
   ],
-  "license": "GPLv3",
+  "license": "GPL-3.0-only",
   "scripts": {
     "serve": "nodemon ./src/main.ts",
     "build": "cross-env NODE_ENV=production tsc -p ./",
+    "build:docker": "docker build -t auproxy-backend .",
+    "start": "node ./dist/src/main.js",
     "lint": "eslint ./src/**/*.ts --fix",
-    "heroku-postbuild": "yarn build && YARN_PRODUCTION=false yarn --cwd auproximity-webui install && yarn --cwd auproximity-webui build && mkdir dist/src/dist && cp -r auproximity-webui/dist dist/src"
+    "build:all": "yarn build && YARN_PRODUCTION=false yarn --cwd auproximity-webui install && yarn --cwd auproximity-webui build && mkdir dist/src/dist && cp -r auproximity-webui/dist dist/src"
```

Comment:
The change from `heroku-postbuild` to `build:all` should be reverted, because we aren't going with the `build:all` route

Reply:
ok fair enough.